Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -468,23 +468,149 @@ public class FeaturevisorInstance {
     }
 
     func evaluateVariation(featureKey: FeatureKey, context: Context = [:]) -> Evaluation {
-        // TODO: write real implementation
-        return Evaluation(
```

Comment:
Kind of liked the previous implementation more, it was... simpler 🤣🤣🤣

Reply:
Yea, I broke KISS rule here for sure :D